eng/scripts/Create-Swagger-APIView.ps1

```diff
@@ -0,0 +1,80 @@
+
+<#
+.DESCRIPTION
+
+
+.PARAMETER ArtifactName
+  Temporary directory for files being processed. Use $(Agent.TempDirectory) on DevOps
+
+#>
+param (
+    [Parameter(Mandatory = $true)]
+    [string]$ArtiFactsStagingDirectory,
+    [Parameter(Mandatory = $true)]
+    [string]$APIViewArtifactsDirectoryName,
+    [Parameter(Mandatory = $true)]
+    [string]$APIViewUri,
+    [Parameter(Mandatory = $true)]
+    [string]$BuildId,
+    [Parameter(Mandatory = $true)]
+    [string]$RepoName,
+    [Parameter(Mandatory = $true)]
+    [string]$PullRequestNumber,
+    [Parameter(Mandatory = $true)]
+    [string]$Language,
+    [Parameter(Mandatory = $true)]
+    [string]$CommitSha
+)
+
+. "$PSScriptRoot/Logging-Functions.ps1"
+
+$apiViewArtifactsDirectory = [System.IO.Path]::Combine($ArtiFactsStagingDirectory, $APIViewArtifactsDirectoryName)
```

Comment:
Should we not try to refactor https://github.com/Azure/azure-sdk-tools/blob/991e31e2493483cf61cd11a5b0703ec6b81b1271/eng/common/scripts/Detect-Api-Changes.ps1 to share some of this logic?

Reply:
Perhaps we can add some more helpers to https://github.com/Azure/azure-sdk-tools/blob/991e31e2493483cf61cd11a5b0703ec6b81b1271/eng/common/scripts/Helpers/ApiView-Helpers.ps1

Reply:
That script depends a lot on `packageProperties` logic which is not implemented for `azure-rest-api-spec`. I tried to do a refactor but felt it made the script overly complex

Reply:
It feels like we could consider creating a helper function to do the APIView call itself. That way we can use common invoking and error handling. Even if for now you just use that new function here with the intent to also consume it elsewhere that might be a good step in the right direction.